recipes/xege/all/conanfile.py

```diff
@@ -0,0 +1,72 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+
+class XegeConan(ConanFile):
+    name = "xege"
+    version = "20.08"
```

Comment:
```suggestion
```

Not needed here, the version will be taken from the `config.yml` file. Without it, this same recipe will serve for other versions as well.